### PR TITLE
fix hallucinations and <noinput> issues in dataset

### PR DESCRIPTION
**Pull request by gururise**
This PR fixes some of the following issues in the cleaned dataset:

1. Fixes several more hallucinations
2. Fixes many \<noinput\> issues that were missed in the original cleaning
3. Splits several instructions that were merged together
4. Fixes a few more wrong answers/unintelligible questions.

### CODE REVIEW

File: alpaca_data_cleaned.json [hunk 1]
```diff
@@ -176075,9 +176075,9 @@
         "output": "Politics"
     },
     {
-        "instruction": "Reorder the following verbs in the order that they occurred in the text.",
-        "input": "ran, held, shouted",
-        "output": "held, shouted, ran"
+        "instruction": "Reorder the following verbs in the order of physical activity required.",
```
Comment (DanielWe2) [inline alpaca_data_cleaned.json:176078]: I find this confusing. You should at least add the expected order in the instructions.

File: alpaca_data_cleaned.json [hunk 2]
```diff
@@ -235086,8 +235086,8 @@
     },
     {
         "instruction": "Generate an input for the following instruction: write a story about a woman exploring a cave",
```
Comment (DanielWe2) [inline alpaca_data_cleaned.json:235088]: I don't really understand that instruction. It should generate an input. But that instruction would not need any input?

File: alpaca_data_cleaned.json [hunk 3]
```diff
@@ -245022,7 +245022,22 @@
     {
         "instruction": "Summarize the historical events underlying the current immigration debate",
```
Comment (DanielWe2) [inline alpaca_data_cleaned.json:245023]: This seems US centric and should maybe rewritten to include that it is about the debate in the USA.

File: alpaca_data_cleaned.json [hunk 4]
```diff
@@ -245022,7 +245022,22 @@
...
+        "output": "Technology has had a profoundly positive effect on daily life. It has increased productivity and given increased access to information, products and services. As a result, it has enabled people to save time and money them to focus on other activities, such as socializing and other leisure activities. Technology has opened up new markets and new ways of doing business and has dramatically reduced communication distances by making communication almost instantaneous."
+    },
+    {
+        "instruction": "Construct a short story",
+        "input": "",
+        "output": "Once upon a time, there was a young lady named Amanda. She was a hardworking student who dreamed of becoming a doctor one day. But life had other plans for her when she met the love of her life, Mark. Mark was a charming and kind-hearted man who swept her off her feet. After dating for a few months, Amanda and Mark decided to tie the knot and get married. From that moment on, they had many happy years together and although Amanda never became a doctor, she was happy with Mark and the life they had built together."
+    },
+    {
+        "instruction": "Classify the following sentence",
```
Comment (DanielWe2) [inline alpaca_data_cleaned.json:245038]: You more clear in what classification you want.

File: alpaca_data_cleaned.json (hunk 3 above)
Comment (gururise) [reply alpaca_data_cleaned.json:245023]: Very true. I didn't notice that, I just split the instructions. Will change to:
Summarize the 3 main historical events leading up to World War I.

File: alpaca_data_cleaned.json (hunk 4 above)
Comment (gururise) [reply alpaca_data_cleaned.json:245038]: Classify the following sentence as fact or fiction.

File: alpaca_data_cleaned.json (hunk 1 above)
Comment (gururise) [reply alpaca_data_cleaned.json:176078]: Good point, will rephrase to: 
        "instruction": "Reorder the following animals in ascending order of height.",
        "input": "giraffe, skunk, lion",
        "output": "skunk, lion, giraffe"

**Review by DanielWe2 — APPROVED** [review]

**Comment by gururise** [comment]
Created a new Repository for a Cleaned Alpaca Dataset:
https://github.com/gururise/AlpacaDataCleaned

Will close this issue.

**Pull request closed without merging**